### DISCUSSION TITLE
Migration from Travis CI to GitHub Workflow(Actions) and Azure Pipelines is finished

### DISCUSSION THREAD

**Discussion by jmalak** [open-watcom/open-watcom-v2#554] [Announcement]
Migration from Travis CI to other similar services is finished.
Now following services are used:
- GitHub Workflow(Actions)
for mirroring open-watcom/open-watcom-v2 repository to SourceForge
for build OW2 Wiki Documentation
- Azure Pipelines
for OW2 pushes build
for OW2 pull request build
for OW2 Coverity Scan data build and uploading
for daily build all Installers and creation of complete OW2 build snapshot

It was migrated to simplify handling of OW build/tests, because on Travis CI are limited resources.
Main limitation is job runtime limit 50 minutes and any log size 4 MB.
Most of OW build jobs exceded these limits.
It was necessary divide jobs to smaller parts and do caching of build pieces to put all together later.
Log files were limited to minimal size that can not contain important information etc.
Now no similar limits for OW build on Azure Pipelines.

**Comment by jmalak** [comment]
Now new release "Latest-CI-build" is add to open-watcom-v2 repository.
It contains only full snapshot without any installers and it is update each time if some change is commited to open-watcom-v2 repository.
Existing release "Current-build" is daily build with all installers as before.

**Comment by XVilka** [comment]
By the way you can use [builds.sr.ht](https://builds.sr.ht) from [SourceHut](https://sourcehut.org) to test [FreeBSD](https://man.sr.ht/builds.sr.ht/compatibility.md#freebsd) and [OpenBSD](https://man.sr.ht/builds.sr.ht/compatibility.md#openbsd) builds.

**Comment by jmalak** [comment]
Thanks for your info.

**Comment by jmalak** [comment]
The "Current-build" script was enhanced to create static release on beginning of each month.
Name is defined by ISO formatted date of build.
By example build from 1-st January 2021 will be named "2021-01-01 Build".
